sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -202,6 +232,9 @@ public synchronized void onActivityPreCreated(
     if (isAllActivityCallbacksAvailable) {
       setColdStart(savedInstanceState);
 
+      // start collecting frame metrics for transaction
+      activityFramesTracker.addActivity(activity);
```

Comment:
Should we start tracking frames before we `setColdStart` or the order here is by design?

Reply:
moved this block 1st thing when the activity methods are called.
the downside is, if tracing is disabled, we will be doing something that is not gonna be used, but if we do that just after all the checks, we might be losing some frames count, lets live with this trade off then and we start collecting the data as early as possible.